src/screens/tabs/Profile/index.js

```diff
@@ -29,6 +29,9 @@ const Profile = () => {
   const modalRef = useRef(null);
   const transactionListRef = useRef(null);
   useScrollToTop(transactionListRef);
+  const { reverseResolvedName } =
+    useSelector(state => state.keyring.currentWallet?.icnsData) || {};
```

Comment:
i think we should avoid the ` || {}` when possible. In this case we could avoid destructuring